### ISSUE TITLE
Update German language version to "Gender-Doppelpunkt"

### ISSUE THREAD

**Issue by hauschke** [vivo-project/VIVO#3719]
**Is your feature request related to a problem? Please describe.**
Current form: BibliothekarIn, which is a gendered form that is not really in use anymore.

**Describe the solution you'd like**
Update the gendered labels to the Gender-Doppelpunkt "Bibliothekar:in"

**Describe alternatives you've considered**
"Gender-Sternchen" maybe?

**Comment by annakasprzik** [comment]
I second the assessment that the form "BibliothekarIn" is antiquated because it is based on the binary gender system. 
The currently prevalent form seems indeed to be "Bibliothekar:in".
Consensus about a suitable form seems to be changing constantly, though (and in my opinion the "in" in ":in" still favours the binary system).
However, if the VIVO community is flexible enough to change the form again every few years then changing it to ":in" now is probably the best move.

**Comment by hauschke** [comment]
> However, if the VIVO community is flexible enough to change the form again every few years then changing it to ":in" now is probably the best move.

Yes, that's probably the way to go. If we don't get any objections, I will try to move this forward.
